wordpress-mobile/WordPress-iOS
# Reader: Remove `ReaderWelcomeBanner`

Ref: pctCYC-195-p2#comment-1144

## Description

Removes the Reader welcome banner.

## Testing

To test:
- Perform a fresh installation of Jetpack
- Login
- Navigate to the reader "Discover" feed
- 🔎 **Verify** the Reader welcome banner does not show

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
